common/djangoapps/xblock_django/models.py

```diff
@@ -40,3 +49,27 @@ def disabled_block_types(cls):
             return ()
 
         return config.disabled_blocks.split()
+
+    @classmethod
+    def deprecated_block_types(cls):
```

Comment:
This should be renamed to `disabled_create_block_types` to match the field name.